ci/test/pipeline.template.yml

```diff
@@ -158,6 +158,19 @@ steps:
             - exit_status: 1
               limit: 2
 
+      - id: helm-charts-publish
+        label: Publish Helm Charts
+        command: bin/ci-builder run stable misc/helm-charts/publish.sh
+        timeout_in_minutes: 10
+        inputs:
+          - "*"
```

Comment:
Would it make sense to change the inputs to `misc/helm-charts` so that this would only run if there are changes to the charts directory?

Reply:
Done